Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Based on https://github.com/DazWilkin/do-apps-rust
+FROM rustlang/rust:nightly-slim as builder
+
+RUN USER=root cargo new --bin mimc-fast
+
+WORKDIR /mimc-fast
+
+COPY ./Cargo.toml ./Cargo.toml
```

Comment:
I can do a pass on this later to cache the cargo deps into a seperate stage

Reply:
Yeah, that's what is doing already